specifications/M1/identity-trust-sts-api.yaml

```diff
@@ -0,0 +1,89 @@
+openapi: 3.0.1
+paths:
+  /token:
+    post:
+      operationId: token
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            encoding:
+              access_token:
+                style: form
+              audience:
+                style: form
+              bearer_access_alias:
+                style: form
+              bearer_access_scope:
+                style: form
+              client_id:
+                style: form
+              client_secret:
+                style: form
+              grant_type:
+                style: form
+            schema:
+              type: object
+              properties:
+                access_token:
+                  type: string
+                  description: VP access token to be added as a claim in the SI token
+                audience:
+                  type: string
+                  description: Audience for the SI token
+                bearer_access_alias:
+                  type: string
+                  description: Alias to be use in the sub of the VP access token (default
+                    is audience)
+                bearer_access_scope:
+                  type: string
+                  description: Scope to be added in the VP access token
+                client_id:
+                  type: string
+                  description: Id of the client requesting an SI token
+                client_secret:
+                  type: string
+                  description: Secret of the client requesting an SI token
+                grant_type:
+                  type: string
+                  description: "Type of grant: must be set to client_credentials"
+              required:
+              - audience
+              - client_id
+              - client_secret
+              - grant_type
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StsTokenResponse'
+          description: The Self-Issued ID token
+        "400":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/StsTokenErrorResponse'
+          description: Invalid Request
+      tags:
+      - Secure Token Service Api
+components:
+  schemas:
+    StsTokenErrorResponse:
+      type: object
+      properties:
+        error:
+          type: string
+        error_description:
+          type: string
+    StsTokenResponse:
+      type: object
+      properties:
+        access_token:
```

Comment:
Should we consider another name for the return value? It might be confusing, as this is actually a self-issued ID token (the wrapper around the access token)

Reply:
@borisrizov-zf  which name the `access_token` property?

Reply:
yes, wouldn't it be more appropriate to call it `si_token`? I know it's splitting hairs at this point, but wanted to throw that in the discussion.

Reply:
It's called `access_token` since we'd want to be compliant with client credential flow and response check here

https://datatracker.ietf.org/doc/html/rfc6749#section-4.4.3

Reply:
Ah I see, ok makes sense.